test/solc/CommandLineParser.cpp

```diff
@@ -168,7 +168,7 @@ BOOST_AUTO_TEST_CASE(cli_mode_options)
 
 		expectedOptions.input.addStdin = true;
 		expectedOptions.input.basePath = "/home/user/";
-		expectedOptions.input.allowedDirectories = {"/tmp", "/home", "project", "../contracts", "", "c", "/usr/lib"};
```

Comment:
not sure about that empty string!

Reply:
It's there because some input paths are just file names, without a directory. It was actually a but that I fixed in #11688.

If you want to tackle this, you should really base your PR on #12007, which has all my changes fixing `--allowed-paths`.

Reply:
I pushed this just a couple of minutes after you comment on the issue :)